test/integration/client_test.cpp

```diff
@@ -175,10 +175,13 @@ TEST_F(ClientServerTest, SendTxWhenStatelessInvalid) {
       "                }]}";
 
   auto doc = iroha::model::converters::stringToJson(json_tx).value();
-  iroha::model::converters::JsonTransactionFactory transactionFactory;
-  auto tx = transactionFactory.deserialize(doc).value();
-  iroha::model::HashProviderImpl hashProvider;
-  tx.tx_hash = hashProvider.get_hash(tx);
+  iroha::model::converters::JsonTransactionFactory jsonTransactionFactory;
+  auto tx = jsonTransactionFactory.deserialize(doc).value();
+  iroha::model::converters::PbTransactionFactory pbTransactionFactory;
+  auto hash = iroha::sha3_256(pbTransactionFactory.serialize(tx)
```

Comment:
That will definitely not a proper way to use crypto stuff. Add todo for writing subfunctions like `hash(const Transaction&)` or `hash(const Block&)`

Reply:
Good point. I added TODO; actual work will require a lot of code changes so it should be done in separate PR I think.

Reply:
Yes, thanks

Reply:
@luckychess please tell me which TODO task to add to backlog 😄